### ISSUE TITLE
Hosting on vercel

### ISSUE THREAD

**Issue by tent4kel** [Podginator/KoboOmnivoreConverter#5]
Is there a technical reason why this would not work on vercel? I understood that they offer server less functions within their free tier:

https://vercel.com/guides/using-express-with-vercel

**Comment by Podginator** [comment]
Yes, and no. So basically at the moment the way that it works is by changing the binary for Kobos main software. So in order to limit this we can only change it to the same, or less bytes. 

I was looking into how to do this in another way, but I haven't had the time. One idea was to just catch the URL when it is being called by the kobo and change it. This would allow you to do that, but I haven't really got around to it yet. 

If you host it, and you can get the name down to a certain size it should work, but you'd have to do quite a lot of fiddling with the internals of the kobo.

**Comment by Podginator** [comment]
Yes, that's exactly right. I just use my raspberry pi because I didn't need to sync outside of the house anyway

**Comment by Podginator** [comment]
Hey! It is now possible to host this on Vercel. I've updated the way the redirect happens, now it doesn't have to have the same bit length. You can update it in the following file: https://github.com/Podginator/KoboOmnivoreConverter/blob/main/hook/pocket.cc#L38

Then follow the instructions here: 
https://github.com/Podginator/KoboOmnivoreConverter/blob/main/hook/README.md

to get it up and running.

**Comment by tent4kel** [comment]
Great! Can't wait to try it!
